Sming/Makefile-project.mk

```diff
@@ -72,6 +72,17 @@ else
       ESP_HOME ?= /opt/esp-open-sdk
       include $(SMING_HOME)/Makefile-linux.mk     
   endif
+  ifeq ($(UNAME),FreeBSD)
+      # Freebsd Detected
+      UNAME := FreeBSD
+
+      # Default SMING_HOME. Can be overriden.
+      SMING_HOME ?= /usr/home/shurik/src/esp8266/Sming/Sming
```

Comment:
What I meant, please set SMING_HOME default to something generic in the general make file, like @anakod suggested and use project overrides or env variables to set it to your environment specifics.


Thank you.

> On Jun 6, 2015, at 8:40 AM, Dmitry Kireev <dmitry@kireev.co> wrote:
> 
> Please use SMING_HOME variable
> 
> Sent from my iPhone
> 
>> On Jun 6, 2015, at 2:03 AM, Alexander V. Ribchansky <notifications@github.com> wrote:
>> 
>> In Sming/Makefile-project.mk:
>> 
>> > @@ -72,6 +72,17 @@ else
>> >        ESP_HOME ?= /opt/esp-open-sdk
>> >        include $(SMING_HOME)/Makefile-linux.mk     
>> >    endif
>> > +  ifeq ($(UNAME),FreeBSD)
>> > +      # Freebsd Detected
>> > +      UNAME := FreeBSD
>> > +
>> > +      # Default SMING_HOME. Can be overriden.
>> > +      SMING_HOME ?= /usr/home/shurik/src/esp8266/Sming/Sming
>> 06 июня 2015 г. 2:56 пользователь "Skurydin Alexey" < notifications@github.com> написал:
>> > + ifeq ($(UNAME),FreeBSD) > + # Freebsd Detected > + UNAME := FreeBSD2 > + > + # Default SMING_HOME. Can be overriden. > + SMING_HOME ?= /usr/home/shurik/src/esp8266/Sming/Sming It is absolute path in your OS? Can it be replaced by relative?
>> Yes, it can replaced by relative path like /usr/local/esp-open-sdk or more relative ~/esp-open-sdk or similar path. Or I miss you question??
>> —
>> Reply to this email directly or view it on GitHub.
>>